packages/concerto-tools/lib/codegen/fromcto/csharp/csharpvisitor.js

```diff
@@ -228,6 +228,24 @@ class CSharpVisitor {
         return null;
     }
 
+    /**
+     * Visitor design pattern
+     * @param {Field} field - the object being visited
+     * @param {Object} parameters  - the parameter
+     * @return {Object} the result of visiting or null
+     * @private
+     */
+    visitScalarField(field, parameters) {
+        let fieldType;
+        let scalarField = field.getScalarField();
+        if (scalarField.getType() === 'String') {
+            if(ModelUtil.removeNamespaceVersionFromFullyQualifiedName(field.getFullyQualifiedTypeName()) === 'concerto.scalar.UUID') {
+                fieldType = 'concerto.scalar.UUID';
+            }
+        }
+        return this.writeField(field.getScalarField(), parameters, fieldType);
```

Comment:
I think you can make this a "one-liner", something like...

```
const fieldType = ModelUtil.removeNamespaceVersionFromFullyQualifiedName(field.getFullyQualifiedTypeName());

return this.writeField(field.getScalarField(), parameters, fieldType === 'concerto.scalar.UUID' ? fieldType : null );
```

Reply:
thanks for the suggestion.